nautobot/apps/config.py

```diff
@@ -0,0 +1,12 @@
+"""Helper code for loading values that may be defined in settings.py/nautobot_config.py *or* in django-constance."""
+
+from constance import config
+from django.conf import settings
+
```

Comment:
Probably should expose `ConstanceConfigItem` class in the `nautobot.apps` namespace, either here in `config.py` or else in `__init__.py` alongside `NautobotAppConfig`.

Reply:
Yep, added.